pkg/graph/schema.graphql

```diff
@@ -808,6 +808,7 @@ type BusinessCase {
 
 """
 The status of a system's IT governence request
+Note - pre-IT Gov v2 only - for IT Gov v2, use SystemIntakeRequesterStatus/SystemIntakeAdminStatus
```

Comment:
Just some naming changes in this comment to match the type names
```suggestion
Note - pre-IT Gov v2 only - for IT Gov v2, use SystemIntakeStatusRequester/SystemIntakeStatusAdmin
```

Reply:
Good catch; updated in https://github.com/CMSgov/easi-app/pull/2108/commits/74b63a81f42637496196fc07bff8a2b0a7efc102.